### ISSUE TITLE
Cannot support LTE_RRC_OTA packet version 15

### ISSUE THREAD

**Issue by zwyuan** [mobile-insight/mobileinsight-core#29]
On Google Pixel 2 XL phone with T-mobile, US (310260), MobileInsight cannot support newest version of the LTE RRC OTA packet and LTE PHY PDSCH Demapper Configuration version

```
11-22 12:59:10.180 13167 13181 I python2.7:  2017-11-22,12:59:10.180132 [b][color=00ff00][INFO][/color] [OnlineMonitor][/b]: Enable collection: LTE_PHY_Inter_RAT_Measurement
11-22 12:59:11.350 13167 13181 I python  : (MI)Unknown LTE RRC OTA packet version: 15
11-22 12:59:12.545 13167 13181 I python  : (MI)Unknown LTE PHY PDSCH Demapper Configuration version: 0x7a
```

**Comment by zwyuan** [comment]
The RRC OTA version is fixed by commit 33b336d.
PHY PDSCH Demapper Configuration to be updated.

**Comment by zwyuan** [comment]
Supported by PR #30, #33, #35, #36, #37, #38
